src/ax/marshaling/message_test.go

```diff
@@ -32,49 +33,102 @@ var _ = Describe("MarshalMessage", func() {
 })
 
 var _ = Describe("UnmarshalMessage", func() {
+
 	message := &messagetest.Message{
 		Value: "<value>",
 	}
-	data, err := proto.Marshal(message)
+
+	nonaxmsg := &messagetest.NonAxMessage{}
+
+	pbdata, err := proto.Marshal(message)
```

Comment:
I think this serialized data should have longer names, since they're used as input for tests and there's a bunch of them now. I'd suggest `axMessageJSON`, `axMessagePB`, `nonAxMessageJSON`, etc...

Reply:
Cool, correcting them...